controllers/gce/controller/controller.go

```diff
@@ -329,21 +337,36 @@ func (lbc *LoadBalancerController) sync(key string) (err error) {
 	if !ingExists {
 		return syncError
 	}
+	ing := obj.(*extensions.Ingress)
```

Comment:
Hmm, the dereference copy may have been on purpose from the original author. Thoughts?

Reply:
hmm I dont see a reason why but am not confident enough yet. Reverted for now. Will look closer and probably change it in a different PR